compte-utilisateur/personnalisation-profil/personnalisation-profil.md

```diff
@@ -8,17 +8,17 @@ keywords:
   - compte
   - tutoriel
 description: Le profil utilisateur sur discord
-contributors: [ichii]
+contributors: [ichii, antoine]
 ---
 ## Les profils d'utilisateur sur Discord
-![Profil](https://i.discord.fr/xfc7.png)
+![Profil](https://i.dfr.gg/xfc7.png)
 
 Le profil d'un utilisateur est unique et possède plusieurs éléments distinctifs:
-- **Vert:** La [bannière de profil](https://discord.fr/wiki/parametres-compte/personnalisation-compte/banniere-profil/) est une image personnalisée ou une couleur.
+- **Vert:** La bannière de profil est une image personnalisée ou une couleur.
```

Comment:
Le **lien était correct** (`/wiki/compte-utilisateur/personnalisation-profil/bannieres-avatars`). Il est positif de garder des redirections vers d'autres pages du wiki (même si c'est dans la même catégorie) ou du blog.

Une alternative au lien serait de placer une [DocCardList](https://docusaurus.io/docs/2.x/sidebar/items#embedding-generated-index-in-doc-page) en bas de la page.

Sur **la formulation**, aussi, c'est l'image et la couleur qui sont personnalisables. La phrase est actuellement tournée d'une manière où l'on peu comprendre que la couleur n'est pas modifiable.